Exercises-3-Basic/src/exercise/Number1.java

```diff
@@ -0,0 +1,21 @@
+package exercise;
+//Write a Java program to sum the values of an array.
```

Comment:
The whole description should be together.
No need newline between the description and class definition.

Reply:
done.